x-pack/legacy/plugins/reporting/public/hacks/job_completion_notifier.js

```diff
@@ -89,6 +89,7 @@ uiModules.get('kibana')
       const downloadReportButton = (
         <EuiButton
           size="s"
```

Comment:
This is my hack to get the URL of the report, vs clicking on it and trying to navigate to that Tab in webdriver, which appears to not be working in headless (just freezes or crashes there). A pro is that this speeds up the tests!

Reply:
This is genius!